### PR TITLE
Add a glossary to the documentation

### PR DESCRIPTION
**Pull request by djee-ms**
Add a new documentation page containing a glossary of common WebRTC
concepts explained.

### CODE REVIEW

**Comment by djee-ms** [comment]
This makes it easier to reference the explanations from the various tutorials rather than repeat them in each tutorial.

image

File: docs/manual/glossary.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+uid: glossary
+---
+
+# Glossary
+
+## D
```
Comment (fibann) [inline docs/manual/glossary.md:7]: nit: do we really need letter sections? There are < 10 entries and just one for (almost) every letter.

File: docs/manual/glossary.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+> See also
+>
+> - [_media track_](#media-track)
+> - [_remote media track_](#remote-media-track)
+
+## M
+
+### Media
+
+The term _media_ refers to either or both of _audio_ and _video_. It does **not** encapsulate [_data channels_](#data-channel).
```
Comment (fibann) [inline docs/manual/glossary.md:28]:
```suggestion
The term _media_ refers to either or both of _audio_ and _video_. It does **not** include [_data channels_](#data-channel).
```
Note sure "encapsulate" is correct here.

File: docs/manual/glossary.md (hunk 1 above)
Comment (djee-ms) [reply docs/manual/glossary.md:7]: I find readability much better. This also prevents the right-side menu from being one long flat list. Glossary generally have per-letter grouping/section too. I'm also hopeful we can add more definitions as needed.

File: docs/manual/glossary.md (hunk 2 above)
Comment (djee-ms) [reply docs/manual/glossary.md]: Fixed.

**Pull request merged**